content/updates/2020-05/2-2-way-associations.md

```diff
@@ -0,0 +1,19 @@
+# Product associations update
+
+## New association type: the 2-way association
```

Comment:
I'm not comfortable with having only one subsection under the main title.
If there is only one new feature regarding the associations in May, I'd suggest removing putting only one main title.

Reply:
@LaureBro, what do you think?

Reply:
OK, I'll make a new PR.